data/players.yaml

```diff
@@ -2858,3 +2858,12 @@
     - '551488'
     voobly:
     - '123226583'
+- name: Valeria Huesca
+  country: mx
+  twitch: https://www.twitch.tv/valeriahuesca
+  discord: https://discord.gg/z7Pvx7g
+  platforms:
+    de:
+    - '410691'
+  youtube: https://www.youtube.com/channel/UCDAwFPixAVPrUogkwhN0iAg
+  team: DarkSidE
```

Comment:
If this is correct you might want to add her to https://github.com/SiegeEngineers/aoc-reference-data/blob/master/data/teams.json as well

Reply:
Is added in #74